meta/main.yml

```diff
@@ -0,0 +1,21 @@
+galaxy_info:
+  role_name: foreman_modules
+  author: The Foreman
+  description: Deploy Foreman Ansible Modules as a Role
+  issue_tracker_url: https://github.com/ericsysmin/ansible-role-foreman-modules/issues
+  license: GPLv3
+  min_ansible_version: 2.0.0
+  github_branch: master
+  platforms:
+  - name: EL
+    versions:
+    - 6
+    - 7
+  - name: Ubuntu
+    versions:
+    - precise
+    - trusty
+    - xenial
+  galaxy_tags:
+    - foreman
+dependencies: []
```

Comment:
Nailgun ?

Reply:
@sean797 are you talking about as a tag, or dependency? In meta/main.yml dependencies are role dependencies which mean other Ansible roles are required for this one. Anything here will search for Galaxy Roles to install at the same time.